examples/aks.preflight.yaml

```diff
@@ -16,9 +16,9 @@ enabled-packages:
 
 # This configures how the results will be reported.
 outputs:
-# writes a human readable doc that can be opened with a browser.
-- type: local
-  path: ./output
+# uploads a json file to Azure Blob Storage
+- type: azblob
+  container: mycontainer
```

Comment:
Possibly also note that the container must already exist.

Reply:
Added to `./docs/configuration.md`.